### PR TITLE
[DOCS-6489] - Add customize process/task list configuration

### PR DESCRIPTION
**Pull request by oliveralfresco2018**

### CODE REVIEW

**Review by sivakumar414ram — DISMISSED** [review]

File: process-automation/latest/develop/index.md [hunk 1]
```diff
@@ -93,6 +88,530 @@ Once the extended Digital Workspace has been fully customized and tested it can
...
+
+  constructor() {}
+
+  ngOnInit() {
+    this.displayValue = this.context?.row?.obj;
+  }
+}
+```
+
+Register the custom components in the [task-list.module.ts](https://github.com/Alfresco/alfresco-apps/blob/9a7c4106fb5befc05bb45e2703bd70c6a7436fb1/libs/content-ee/process-services-cloud-extension/src/lib/features/task-list/task-list.module.ts){:target="_blank"} file. For more on how to register a custom component see [Registration](https://github.com/Alfresco/alfresco-content-app/blob/develop/docs/extending/registration.md){:target="_blank"}.
```
Comment (anxumalo) [inline process-automation/latest/develop/index.md:313]: The first link points to a specific commit - should it link to the default branch instead?

File: process-automation/latest/develop/index.md [hunk 2]
```diff
@@ -93,6 +88,530 @@ Once the extended Digital Workspace has been fully customized and tested it can
...
+```typescript
+  this.columns = this.extensions.getProcessListPreset('my-process-presets');
+```
+
+![Process list with new columns]({% link process-automation/images/process-list-additional-columns.png %})
+
+### Add a column in the process list using a custom template
+
+To display a process list with a custom column template you first need to create a custom component.
+For example, to create custom templates for the process name, status properties, edit the `process-services-cloud.extension.json` file in the following way.
```
Comment (anxumalo) [inline process-automation/latest/develop/index.md:487]:
```suggestion
For example, to create custom templates for the process name, and status properties, edit the `process-services-cloud.extension.json` file in the following way.
```

File: process-automation/latest/develop/index.md [hunk 3]
```diff
@@ -93,6 +88,530 @@ Once the extended Digital Workspace has been fully customized and tested it can
...
+}
+```
+
+![Task list with default columns]({% link process-automation/images/task-list-default-columns.png %})
+
+### Add a column in the task list using a task property
+
+To display the task list with new columns, edit the `process-services-cloud.extension.json` file, and insert an entry into the `features.taskList.presets.default` section.
+
+Task list with new columns.
```
Comment (anxumalo) [inline process-automation/latest/develop/index.md:117]: I don't think this statement is needed - it's already mentioned in the previous sentence.

File: process-automation/latest/develop/index.md [hunk 4]
```diff
@@ -93,6 +88,530 @@ Once the extended Digital Workspace has been fully customized and tested it can
...
+}
+```
+
+![Process list with default columns]({% link process-automation/images/process-list-default-columns.png %})
+
+### Add a column in the process list using a process instance property
+
+To display the process list with new columns, edit the `process-services-cloud.extension.json` file, and insert an entry into the `features.processList.presets.default` section.
+
+Process list with new columns.
```
Comment (anxumalo) [inline process-automation/latest/develop/index.md:401]: Similar to an earlier comment, I don't think this statement is needed - it's already mentioned in the previous sentence.

File: process-automation/latest/develop/index.md [hunk 5]
```diff
@@ -93,6 +88,530 @@ Once the extended Digital Workspace has been fully customized and tested it can
     * The environment the application is deployed in.
     * When the application should be updated with the new interface.
 
+### Extend task list with custom columns
+
+The Digital Workspace provides a default group of columns for your task list. The columns can be changed and configured in the `process-services-cloud.extension.json` file of the [Process Services Cloud extension library](https://github.com/Alfresco/alfresco-apps/tree/develop/libs/content-ee/process-services-cloud-extension){:target="_blank"}.
+
+Task list with default columns preset.
```
Comment (anxumalo) [inline process-automation/latest/develop/index.md:95]: Feel free to change the wording:
```suggestion
Here is an example of a task list with default columns preset:
```

**Review by anxumalo — COMMENTED** [review]
LGTM - built OK - some minor comments for your consideration

File: process-automation/latest/develop/index.md (hunk 3 above)
Comment (oliveralfresco2018) [reply process-automation/latest/develop/index.md:117]: Changed

File: process-automation/latest/develop/index.md (hunk 4 above)
Comment (oliveralfresco2018) [reply process-automation/latest/develop/index.md:401]: changed

File: process-automation/latest/develop/index.md (hunk 1 above)
Comment (oliveralfresco2018) [reply process-automation/latest/develop/index.md:313]: I confirmed that with Ankur yesterday he seems ok with where those links are going. Cheers.

**Review by anxumalo — APPROVED** [review]

**Pull request merged**